### PR TITLE
Fix enforcer rules during full profile build

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@psiroky please take a look.

File: jbpm-wb-showcase/pom.xml [hunk 1]
```diff
@@ -392,6 +392,10 @@
           <artifactId>slf4j-jdk14</artifactId>
           <groupId>org.slf4j</groupId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.geronimo.specs</groupId>
+          <artifactId>geronimo-activation_1.1_spec</artifactId>
+        </exclusion>
```
Comment (psiroky) [inline jbpm-wb-showcase/pom.xml]: This should not be (hopefully) needed after we merge https://github.com/droolsjbpm/jbpm-designer/pull/556

File: jbpm-wb-integration/jbpm-wb-integration-backend/src/test/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProvisioningIT.java [hunk 2]
```diff
@@ -57,8 +57,7 @@ public static WebArchive create() {
                         "org.guvnor:guvnor-ala-registry-local",
                         "org.guvnor:guvnor-ala-services-api",
                         "org.guvnor:guvnor-ala-build-maven",
-                        "org.uberfire:uberfire-io",
-                        "org.sonatype.sisu.inject:guice-servlet"
```
Comment (psiroky) [inline jbpm-wb-integration/jbpm-wb-integration-backend/src/test/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProvisioningIT.java]: Should we replace this with `com.google.inject.extensions:guice-servlet`?

Comment (cristianonicolai) [inline jbpm-wb-integration/jbpm-wb-integration-backend/src/test/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProvisioningIT.java]: @psiroky Seems like it is working fine as it gets resolved as a transitive dependency.

**Pull request merged**